<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MumeClockWidget</class>
 <widget class="QWidget" name="MumeClockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>148</width>
    <height>22</height>
   </rect>
  </property>
  <property name="minimumWidth">
   <number>113</number>
  </property>
  <property name="windowTitle">
   <string>Mume Clock</string>
  </property>
  <property name="toolTip">
   <string>Mume Clock</string>
  </property>
  <property name="statusTip">
   <string>Current time in Mume</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="alignment">
    <enum>Qt::AlignLeft</enum>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="moonPhaseLabel">
       <property name="text">
        <string>Moon Phase</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="seasonLabel">
       <property name="text">
        <string>Season</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="timeLabel">
       <property name="text">
        <string>Clock not set</string>
       </property>
       <property name="minimumWidth">
        <number>40</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
